spec/controllers/account/profile_controller_spec.rb

```diff
@@ -0,0 +1,46 @@
+require 'rails_helper'
+
+RSpec.describe Account::ProfileController, type: :controller do
```

Comment:
render views

Reply:
renders views added globally I believe in `support` folder